code/game/turfs/turf.dm

```diff
@@ -73,6 +73,14 @@ GLOBAL_LIST_EMPTY(station_turfs)
 	/// See __DEFINES/construction.dm for RCD_MEMORY_*.
 	var/rcd_memory
 
+	/// Minimum level of rust_strength needed to rust this tile
+	/// 1: Basic iron
+	/// 2: Reinforced floors/walls, unlocked early
+	/// 3: Titanium/Plastitanium, unlocked late
+	/// 4: Most other materials, unlocked at ascention
+	/// 5: Completely immune to being rusted
+	var/rust_resistance = 4
```

Comment:
These should probably be defines.
These are just ideas, if you have better ones feel free to use them:
`RUST_RESISTANCE_BASIC`
`RUST_RESISTANCE_REINFORCED`
`RUST_RESISTANCE_TITANIUM`
`RUST_RESISTANCE_ORGANIC`
`RUST_RESISTANCE_ABSOLUTE`